index.js

```diff
@@ -32,18 +32,17 @@ const defaultApps = [
 process.on('unhandledRejection', logger.error.bind(logger))
 
 module.exports = (options = {}) => {
-  const webhook = createWebhook({path: options.webhookPath || '/', secret: options.secret || 'development'})
```

Comment:
the `webhook` instance is exposed as part of the `probot` instance API, so strictly speaking this is a breaking change as the `probot.webhook`’s API has changed. But I haven’t seen it being used anywhere? Can you recall why you exposed it? @bkeepers 

https://github.com/probot/probot/blob/c5a0c5956c529577a8019bd3fa0cbab57282d13f/lib/index.js#L81-L93